azuresdk/README.md

```diff
@@ -49,12 +49,13 @@ Following environment variables needs to be set (automatically set via workloadi
 
 ### Azure Cloud/Environment support
 
-| `AZURE_ENVIRONMENT`    | Description                                                                                  |
-|------------------------|----------------------------------------------------------------------------------------------|
-| `AzurePublicCloud`     | Default Azure cloud, using https://portal.azure.com                                          |
-| `AzureChinaCloud`      | Azure cloud in China, using https://porta.azure.cn                                           |
-| `AzureGovernmentCloud` | US Government Azure cloud                                                                    |
-| `AzurePrivateCloud`    | Private on-premise installation of Azure Cloud, needs additional configuration for endpoints |
```

Comment:
I updated this table to better match the logic in the below switch statement where we match on "aliased" versions of the envvar. 

https://github.com/dtrejod/go-common/blob/dt/azure-secret/azuresdk/cloudconfig/cloudconfig.go#L21